cpp/src/plasma/CMakeLists.txt

```diff
@@ -59,8 +63,6 @@ add_custom_command(
 
 add_custom_target(gen_plasma_fbs DEPENDS ${PLASMA_FBS_OUTPUT_FILES})
 
-add_dependencies(gen_plasma_fbs flatbuffers_ep)
```

Comment:
Does putting the dependency before the creation of the target cause any problems?